src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -229,4 +229,45 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('ip_address', help='IPv4 address or CIDR range.')
 
     with self.argument_context('acr check-health') as c:
-        c.argument('ignore_errors', options_list=['--ignore-errors'], help='Ignore errors, displaying them only at the final', action='store_true', required=False)
+        c.argument('ignore_errors', options_list=['--ignore-errors'], help='Ignore errors, not stopping at first error', action='store_true', required=False)
+
+    with self.argument_context('acr scope-map') as c:
+        c.argument('registry_name', options_list=['--registryName', '-r'])
+        c.argument('description', options_list=['--description'], help='Description for the scope map. Max of 140 characters.', required=False)
+        c.argument('add_repository', nargs='+', options_list=['--add'], help='Repositories to be added. Add tuples (repository;action1,action2,...) separated by space.', required=False)
+        c.argument('remove_repository', nargs='+', options_list=['--remove'], help='Repositories to be removed. Add tuples (repository;action1,action2,...) separated by space.', required=False)
+        c.argument('reset_map', options_list=['--reset'], help='Flag indicating whether scope map should be reset (erase all configurations)', action='store_true', required=False)
+        c.argument('scope_map_name', options_list=['--name', '-n'], help='The name of the target scope map.', required=True)
+
+    with self.argument_context('acr scope-map create') as c:
+        c.argument('add_repository', nargs='+', options_list=['--add'], help='Repositories to be added. Add tuples (repository;action1,action2,...) separated by space.', required=True)
+
+    with self.argument_context('acr token') as c:
+        c.argument('registry_name', options_list=['--registryName', '-r'])
+        c.argument('token_name', options_list=['--name', '-n'], help='The name of the target token.', required=True)
+        c.argument('scope_map_name', options_list=['--scope-map'], help='The name of the scope map associated with the token', required=False)
+        c.argument('status', options_list=['--status'], help='The status of the token. Allowed values are "enabled" or "disabled".', required=False)
```

Comment:
If there is an enum for different actions thats can be added, I would recommend adding: `arg_type=get_enum_type(<MyEnum>)`

Please scan this for possible enum values.